src/common.css

```
@@ -102,13 +102,15 @@
     flex-flow: column;
     justify-content: center;
     align-items: center;
-    position: absolute;
+    position: fixed;
     left: 0;
     top: 0;
+    right: 0;
+    bottom: 0;
     width: 100vw;
     max-width: 100vw;
```

You changed `max-height` to `min-height`. For `max-width` it was not changed.
```suggestion
    min-width: 100vw;
```